hal/src/main/native/athena/ctre/CtreCanNode.cpp

```diff
@@ -68,17 +68,13 @@ void CtreCanNode::UnregisterTx(uint32_t arbId)
 		_txJobs.erase(iter);
 	}
 }
-timespec diff(const timespec & start, const timespec & end)
-{
-	timespec temp;
-	if ((end.tv_nsec-start.tv_nsec)<0) {
-		temp.tv_sec = end.tv_sec-start.tv_sec-1;
-		temp.tv_nsec = 1000000000+end.tv_nsec-start.tv_nsec;
-	} else {
-		temp.tv_sec = end.tv_sec-start.tv_sec;
-		temp.tv_nsec = end.tv_nsec-start.tv_nsec;
-	}
-	return temp;
+static int64_t GetTimeMs() {

```

Comment:
Could we use wpi::Now() instead?

Reply:
I switched this to use the FPGA clock directly.